go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/ProtonMail/go-crypto v1.0.0
 	github.com/atotto/clipboard v0.1.4
 	github.com/blang/semver/v4 v4.0.0
-	github.com/caspr-io/yamlpath v0.0.0-20200722075116-502e8d113a9b
+	github.com/caspr-io/yamlpath v0.0.1
```

Comment:
Notice that the v0.0.1 has no licence and is missing ~10 commits compared to the latest master.
Maybe we should pin the latest one?